Fix a runtime error on some devices. Without this patch, code from the icu libraries might fail to run with a message "internal error", and ldd will output "not a dynamic executable" on them.
--- a/source/config/mh-linux	2020-04-22 19:49:10.000000000 +0200
+++ b/source/config/mh-linux	2020-09-05 17:58:05.635014182 +0200
@@ -23,7 +23,7 @@
 LD_RPATH_PRE = -Wl,-rpath,
 
 ## These are the library specific LDFLAGS
-LDFLAGSICUDT=-nodefaultlibs -nostdlib
+#LDFLAGSICUDT=-nodefaultlibs -nostdlib
 
 ## Compiler switch to embed a library name
 # The initial tab in the next line is to prevent icu-config from reading it.
